lib/ansible/modules/system/iptables_raw.py

```diff
@@ -0,0 +1,556 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2017, Tyler Gates <tgates81@gmail.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = """
+---
+module: iptables_raw
+short_description: Modify runtime iptables rules
+requirements: []
+version_added: "2.7"
+author: Tyler Gates <tgates81@gmail.com>
+description:
+  - Iptables manages the system firewall backed by netfilter in the kernel.
+  - This module allows one to manage the chains and rules in a way that is
+    consistent to the existing iptables cli/saved rules syntax.
+  - This, along with task ordering and robust chain and rule actions, allows for
+    great flexibility and power.
+  - Complex rules can be achieved through a single task, eliminating potential
+    mid-flight lockout induced by multiple module calls while building the
+    rules.
+notes:
+  - chain_* and rule_* options require a list of dictionary values. The key
+    should designate the task number and the value will be the rule or chain.
+options:
+  ipv:
+    description:
+      - Use protocol 4 or 6 when creating rules.
+    required: false
+    default: "4"
+    choices: [ "4", "6" ]
+    version_added: "2.7"
```

Comment:
As this is a new module, we only need the top level `version_added`, no need for it per option.
```suggestion

```